### PR TITLE
fixes for issues #323 and #345

### PR DESCRIPTION
**Pull request by lpinne**
Hi Meike,

small fixes for 
SLES4SAP-hana-sr-guide-PerfOpt-15
and
SAP-S4HA10-setup-simplemount-sle15

Regards,
Lars

### CODE REVIEW

File: adoc/SLES4SAP-hana-sr-guide-PerfOpt-15.adoc [hunk 1]
```diff
@@ -1558,19 +1558,13 @@ Define the watchdog device. It is mandatory to use a watchdog. SBD does not work
...
-you production environment.
-
-The parameter must also be aligned with the pacameker cluster property stonith-watchdog-timeout.
-The property stonith-watchdog-timeout must be set >= the value of SBD_WATCHDOG_TIMEOUT.
-
-// from the product documentation - copied 2020-02-05
-If you set `stonith-watchdog-timeout` to a negative value, Pacemaker will automatically calculate this timeout and set it to twice the value of `SBD_WATCHDOG_TIMEOUT` starting with {sleha} {prodNr}.
+|SBD_WATCHDOG_TIMEOUT a|
+This parameter is used with diskless SBD.
+It defines the timeout, in seconds, the watchdog will wait before paniking the
```
Comment (chabowski) [inline adoc/SLES4SAP-hana-sr-guide-PerfOpt-15.adoc:1563]: ... in seconds, watchdog will wait before panicking ...

File: adoc/SLES4SAP-hana-sr-guide-costopt-15.adoc [hunk 2]
```diff
@@ -1664,18 +1664,24 @@ See the manual pages sbd(8) and stonith_sbd(7) for details.
...
-up a watchdog.
-
-|SBD_STARTMODE="always" a|
+|SBD_WATCHDOG_DEV a|
+Define the watchdog device. It is mandatory to use a watchdog. SBD does not
+work reliable without watchdog. Refer to the {slsa} manual and SUSE TIDs 7016880 for setting up a watchdog.
+
+|SBD_WATCHDOG_TIMEOUT a|
+This parameter is used with diskless SBD.
+It defines the timeout, in seconds, the watchdog will wait before paniking the
```
Comment (chabowski) [inline adoc/SLES4SAP-hana-sr-guide-costopt-15.adoc:1673]: ... in seconds, watchdog will wait before panicking the node ...

**Comment by lpinne** [comment]
Hi Meike,

all suggested changes are fine with me.
Would you like to apply them?

Regards,
Lars

**Comment by chabowski** [comment]
Hi Lars, yes I can do so - but I will take it from here, means merge your PR first and apply the changes and push them.

**Pull request merged**